docs/source/how_tos/visualize_data_with_tableau.rst

```diff
@@ -0,0 +1,46 @@
+***************************
+Visualize Data with Tableau
+***************************
+`Tableau <https://www.tableau.com/>`_ is a nice commercial tool for exploring and visualizing
+tabular data. Licenses are available to NRELians.
```

Comment:
I'd either delete the sentence about licenses being available or make it more actionable. (I.e., if I'm an NREL staff member and just learning this now, what should I do about it? Can I go to the ITS page and download it?)